src/interactors/interactors.test.js

```diff
@@ -29,4 +30,18 @@ describe('Interactors', () => {
     await registerDevice.execute(mockThing);
     expect(cloudMocks.mockAddDevice).toHaveBeenCalled();
   });
+
+  test('unregisterDevice: should execute interactor correctly when unregister a thing from cloud', async () => {
```

Comment:
Same as before (even if this isn't the correct way, it is understandable, programatically speaking, as well as it fits your naming conventions for tests) :point_down: 

```suggestion
  test('unregisterDevice: should execute interactor correctly when unregistering a thing from cloud', async () => {
```

Reply:
Done!